config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  get 'users/:user_id/posts(.:format)', to: 'posts#index'
+  get 'users/:user_id/posts/:id(.:format)', to: 'posts#show'
+  get 'users/(.:format)', to: 'users#index'
+  get 'users/:id(.:format)', to: 'users#show'
```

Comment:
## [OPTIONAL]

- [ ] Kindly note that there is a simplified syntax that can refactor your code. Check here :point_down: 

```rb
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```